### ISSUE TITLE
How to test?

### ISSUE THREAD

**Issue by systemlevel** [codejamninja/react-native-exposure-notification#1]
Hi is there a way to initiate test notifications?

**Comment by clayrisser** [comment]
Yes, it's possible, but I will need to add that into it. I would be happy to add it, but unfortunately I do not have access to the official api. I built this based on a published interface by Google. If you have access or know anyone who has access to the official api, let me know I would be happy to see what I could do to help.

**Comment by Gustavo-Kuze** [comment]
@codejamninja So how are we going to be sure It actually worked after installation?

**Comment by clayrisser** [comment]
Oh it’s not complete because I need to have access to the full API to complete it. Right now it’s just a starting point. Currently when you call the functions it just triggers a toast popup.

What you can do is replace the toast with the specific method binding. Only someone who has access to the api will be able to do this. That is why I keep requesting access from a team that has access.

If you fill in the methods, it would be much appreciated if you would kindly submit it as a pull request. Again you could alternatively give me access to the API and I would be glad to help.

Here is the file where the binding will go.

https://github.com/codejamninja/react-native-exposure-notification/blob/master/android/src/main/java/com/reactlibraryexposurenotification/ExposureNotificationModule.java

**Comment by antonioreyna** [comment]
Hello, so this package is not ready to use the exposure api?

**Comment by clayrisser** [comment]
@antonioreyna that is correct, because I do not have access to the full exposure api. However, I'm hoping it gives teams who have access a head start. If anyone has access to the full api, pull requests would be much appreciated. If you would like me to walk you through the code, let me know.
